src/location-picker/location-picker.component.html

```diff
@@ -13,6 +13,7 @@
             (select)="onSelect($event)">
             <ng-template let-item>
                 <span [innerHTML]="formatLabel(item)"></span>
+                <small class="m-contact-picker__detail u-text-light" [innerHTML]="item.layer"></small>

```

Comment:
m-contact-picker__detail won't work, should be m-location-picker__detail and needs to be added to the stylesheet. Also, is item.layer always set? Perhaps to be on the safe side there should be an *ngIf="item.layer" in there.

Reply:
Fixed. Added to stylesheet as well.